session.go

```diff
@@ -539,6 +539,12 @@ runLoop:
 			s.handleHandshakeComplete()
 		}
 
+		select {
```

Comment:
Comment?

Reply:
Added a comment. Also moved this block up to the `case` that's handling received packets.